cmd/runtimetest/main.go

```diff
@@ -260,6 +271,27 @@ func validateDefaultFS(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateDefaultDevices(spec *rspec.Spec) error {
+	logrus.Debugf("validating linux default devices")
+
+	for _, device := range defaultDevices {
+		fi, err := os.Stat(device)
+		if err != nil {
+			if os.IsNotExist(err) {
+				return fmt.Errorf("Device %v must exist but not", device)
+			} else {
+				return err
+			}
+		} else {
+			if fi.Mode() != os.ModeDevice {
+				return fmt.Errorf("file %v expected is a device but not", device)
```

Comment:
This looks like a reasonable first level of check.  It would be nice to confirm that the device major/minors also match the [spec requirements], but I'm ok punting that to a follow-up PR.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config-linux.md#default-devices